lib/module.js

```diff
@@ -7,7 +7,7 @@ module.exports = function (moduleOptions) {
   }
 
   if (!options.recartSiteId) {
-    console.error('RecartSiteId is required on module options')
+    console.error('RecartSiteId is required in module options')
```

Comment:
I was thinking about reporting this error too, but I don't think this would be very useful for us.

The developer who installs our module will see this error on the console when testing and needs to fix it. 
There's nothing we could do about it.